diff --git a/dist/bin/loadEnv.js b/dist/bin/loadEnv.js
index b61ecc1684f0b686fa8bbcefa68b5ed825c09420..9b044e812e5d86c94add04ffa5666acc9defcfde 100644
--- a/dist/bin/loadEnv.js
+++ b/dist/bin/loadEnv.js
@@ -1,6 +1,6 @@
-import nextEnvImport from '@next/env';
+import * as nextEnvModule from '@next/env';
 import { findUpSync } from '../utilities/findUp.js';
-const { loadEnvConfig } = nextEnvImport;
+const loadEnvConfig = nextEnvModule.loadEnvConfig || (nextEnvModule.default && nextEnvModule.default.loadEnvConfig);
 /**
  * Try to find user's env files and load it. Uses the same algorithm next.js uses to parse env files, meaning this also supports .env.local, .env.development, .env.production, etc.
  */ export function loadEnv(path) {
